src/Kinepolis/DateParser.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Kinepolis;
+
+interface DateParser
```

Comment:
There is no real benefit to this being an interface, it's better to make the full parser an interface.

Reply:
After discussing this, we decided all the Parsers need interfaces, both the main & the subParsers.